service/src/main/kotlin/io/provenance/api/domain/usecase/provenance/account/GetSigner.kt

```diff
@@ -22,25 +17,17 @@ class GetSigner(
     private val entityManager: EntityManager,
     private val provenanceProperties: ProvenanceProperties,
 ) : AbstractUseCase<GetSignerRequest, Signer>() {
-    override suspend fun execute(args: GetSignerRequest): Signer {
-        val originator = entityManager.getEntity(KeyManagementConfigWrapper(args.uuid.toString(), args.account.keyManagementConfig))
+    override suspend fun execute(args: GetSignerRequest): Signer =
+        entityManager.getEntity(KeyManagementConfigWrapper(args.uuid.toString(), args.account.keyManagementConfig)).let {
```

Comment:
Nit: could you rename `it` here so this is easier to follow?

Reply:
done